sdks/java/extensions/euphoria/euphoria-core/build.gradle

```diff
@@ -21,10 +21,31 @@ applyJavaNature(enableFindbugs: false)
 
 description = "Apache Beam :: SDKs :: Java :: Extensions :: Euphoria Java 8 DSL :: Core"
 
+ext {
+    kryoVersion = '2.24.0'
+}
+
 dependencies {
+    shadow project(path: ":beam-sdks-java-core", configuration: "shadow")
+    compile "com.esotericsoftware.kryo:kryo:${kryoVersion}"
     shadow library.java.guava
-    shadow library.java.slf4j_api
     compileOnly library.java.findbugs_jsr305
     testCompile library.java.mockito_core
     testCompile project(path: ':beam-sdks-java-extensions-euphoria-testing')
-}
\ No newline at end of file
+    shadow 'com.google.code.findbugs:annotations:3.0.+'
+    testCompile library.java.slf4j_api
+    testCompile library.java.hamcrest_core
+    testCompile library.java.slf4j_simple
+    testCompile library.java.mockito_core
+    testCompile project(path: ":beam-sdks-java-core", configuration: "testArtifact")
```

Comment:
You should be relying on the `shadowTest` configuration and not `testArtifact`.

This will remove the need for you to define the `testArtifact` in `sdks/java/core`

Reply:
The suggested change was made in 
68fa362. See https://github.com/apache/beam/pull/5783 for more context. Thanks.